api/app/tests/apps/api/v1/test_filters.py

```diff
@@ -1145,3 +1145,35 @@ def test_filter_assigned_user_email(self):
         result_ids = self._request_filter_signals({'assigned_user_email': 'null'})
         self.assertEqual(1, len(result_ids))
         self.assertTrue(self.signal0.id in result_ids)
+
+
+class TestReporterEmailFilter(SignalsBaseApiTestCase):
```

Comment:
Maybe add a test where we see that signals are selected and after they are anonymised no longer appear in the filter

Reply:
added this test